tardis/energy_input/gamma_ray_packet_source.py

```diff
@@ -487,34 +490,57 @@ class GammaRayPacketSource(BasePacketSource):
     def __init__(
         self,
         packet_energy,
-        gamma_ray_lines,
+        isotope_decay_df,
         positronium_fraction,
         inner_velocities,
         outer_velocities,
         inv_volume_time,
         times,
-        energy_df_rows,
         effective_times,
         taus,
         parents,
-        average_positron_energies,
-        average_power_per_mass,
         **kwargs,
     ):
+
+        """
+        New Gamma ray packet source class
+
+        Parameters
+        ----------
+
+        packet_energy : float
+            Energy of the gamma ray packet
+        isotope_decay_df : pd.DataFrame
+            DataFrame of isotope decay data
+        positronium_fraction : float
+            Fraction of positrons that form positronium
+        inner_velocities : array
+            Array of inner shell velocities
+        outer_velocities : array
+            Array of outer shell velocities
+        inv_volume_time : array
+            Array of inverse volume times (please explain)
+        times : array
+            Array of time steps
+        effective_times : array
+            Array of effective time steps
+        taus : dict
+            Dictionary of isotope mean lifetimes in seconds
```

Comment:
This variable name is a bit ambiguous since tau I think we mostly use tau for optical depth in tardis. Could this just be renamed to mean_isotope_lifetime or something like that?

Reply:
Will possibly change in a separate PR.